jruby-gradle-base-plugin/src/main/groovy/com/github/jrubygradle/JRubyPlugin.groovy

```diff
@@ -52,4 +61,57 @@ class JRubyPlugin implements Plugin<Project> {
             dependsOn project.tasks.jrubyPrepare
         }
     }
+
+    // taken from https://github.com/Ullink/gradle-repositories-plugin/blob/master/src/main/groovy/com/ullink/RepositoriesPlugin.groovy
+    boolean setupRubygemsRepositories(Project project) {
+        if (!project.repositories.metaClass.respondsTo(project.repositories, 'rubygems', String, Object)) {
+            project.logger.debug 'Adding rubygems(String?) method to project RepositoryHandler'
+            project.repositories.metaClass.rubygems << { String repoUrl = null ->
+                if (repoUrl == null) repoUrl = 'https://rubygems.org'
```

Comment:
Twe things here:

1. Please make the string constant an actual string constant in the class
1. You can use the [elvis operator](http://docs.groovy-lang.org/latest/html/documentation/index.html#_elvis_operator) to keep this one line without the ugly if statement, e.g.: `repoUrl = repoUrl ?: DEFAULT_RUBYGEMS_URL`